src/JustSaying.Extensions.DependencyInjection.Microsoft/JustSaying.Extensions.DependencyInjection.Microsoft.csproj

```diff
@@ -4,15 +4,19 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <IsShipping>true</IsShipping>
     <RootNamespace>Microsoft.Extensions.DependencyInjection</RootNamespace>
-    <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net461;net8.0</TargetFrameworks>
   </PropertyGroup>
   <ItemGroup>
     <ProjectReference Include="..\JustSaying\JustSaying.csproj" />
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" />
+    <PackageReference Include="Microsoft.Extensions.Options" Condition="$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)','net8.0'))" />
   </ItemGroup>
   <ItemGroup>
     <Using Remove="System.Net.Http" />
   </ItemGroup>
+  <PropertyGroup Condition="$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)','net8.0'))">
+    <IsAotCompatible>true</IsAotCompatible>
+  </PropertyGroup>
```

Comment:
Unless it turns out some of them are redundant, I'd add [all these](https://github.com/App-vNext/Polly/blob/a6b9b33d0ea77c2a559cd58be34e51450b9012c2/src/Polly.Core/Polly.Core.csproj#L14-L20) rather than just this one.

You could potentially put it into the `Directory.Build.props` file and have it turned on automatically for all the library projects once you've gotten them all to 0 warnings.

Reply:
It looks like they all become redundant if you have `<IsAotCompatible>`

[EnableSingleFileAnalyzer, EnableAotAnalyzer](https://github.com/dotnet/sdk/blob/8f8c41a31fe753bed37e09492147579280b8ee9d/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.Analyzers.targets#L76) and [IsTrimmable](https://github.com/dotnet/sdk/blob/8f8c41a31fe753bed37e09492147579280b8ee9d/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Publish.targets#L24) are enabled if IsAotCompatible is enabled and [EnableTrimAnalyzer](https://github.com/dotnet/sdk/blob/8f8c41a31fe753bed37e09492147579280b8ee9d/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.Analyzers.targets#L84) is enabled if IsTrimmable is enabled.